configs/samples/extensions.conf.sample

```diff
@@ -828,3 +828,38 @@ exten => _X.,40000(ani),NoOp(ANI: ${EXTEN})
 ; "core show functions" will list all dialplan functions
 ; "core show function <COMMAND>" will show you more information about
 ; one function. Remember that function names are UPPER CASE.
+
+; Examples using PJSIPNotify application.
+;[generate-notify]
+;
+; Send a NOTIFY with the following headers inside the SIP dialog for the current channel:
+;
+;	Event: Test
+;	Data: Fun
+;
+;exten => 6880,1,noOp()
+;    same => n,Answer()
+;    same => n,PJSIPNotify(,&Event=Test&Data=Fun)
```

Comment:
The code looks for "Content" not "Data".

Reply:
Again with the "Data" :)
See my earlier comment about "Data" being confusing.  Should be "X-Data".